web/src/scripts/actions/tabActions.js

```diff
@@ -75,6 +75,51 @@ export const closeAllTabs = (containerId) => async (dispatch) => {
   dispatch({type: at.CLOSE_ALL_TABS, payload: {containerId}})
 }
 
+export const focusAdjacentTab = (containerId, iterator, options = {}) => async (dispatch, getState) => {
+  const tabs = getState().ui.tabs
+  const { groups, focusedGroupIndex } = _.get(tabs, `${containerId}`)
+  const focusedGroup = groups[focusedGroupIndex]
+  const focusedTabIndex = focusedGroup.tabIds.indexOf(focusedGroup.focusedTabId)
+  const newIndex = iterator(focusedTabIndex)
+  if (newIndex >= 0 && newIndex < focusedGroup.tabIds.length) {
+    dispatch(focusTab(containerId, focusedGroup.tabIds[newIndex], focusedGroupIndex))
+    return
+  }
+  if (options.nowrap) return
```

Comment:
I think the need for `nowrap` here is a symptom of having 2 use cases (adjacent tab, specific index) combined in one action. Let's split in 2 to remove it